packages/core/src/components/Overlay/index.tsx

```diff
@@ -46,12 +46,11 @@ export default class Overlay extends React.PureComponent<Props, State> {
       this.rafHandle = requestAnimationFrame(() => {
         // getBoundingClientRect casues a reflow
         const { x, y } = current.getBoundingClientRect() as DOMRect;
-        if (x !== this.state.x) {
-          this.setState({ x });
-        }
 
-        if (y !== this.state.y) {
-          this.setState({ y });
+        if (x !== this.state.x || y !== this.state.y) {
```

Comment:
adding this second rAF here caused a regression where the tooltip is positioned at 0,0 for a frame or two then at its correct position 

![image](https://user-images.githubusercontent.com/2336595/71328453-45ba7f80-24cc-11ea-846d-17ef80b950ad.png)

![image](https://user-images.githubusercontent.com/2336595/71328460-671b6b80-24cc-11ea-9f41-3eaac025f188.png)

Reply:
I think ^^ could be fixed if we didn't default x and y to `0` or tracked when they are set with a flag. 

```jsx
state = { x: null, y: null }
...
// in render
{open && x !== null && y !== null && (
```